### DISCUSSION TITLE
Feature Request: Direct Tab Navigation with "Ctrl + [1,2,3,...]"

### DISCUSSION THREAD

**Discussion by gustavomarchesim** [zen-browser/desktop#1272] [Ideas]
Currently, the software allows switching between open tabs using the "Ctrl + Tab" shortcut, which cycles through tabs sequentially. While this is useful, it would be great to also implement a direct navigation feature using "Ctrl + [1,2,3,...]" on Windows/Linux (or "Command + [1,2,3,...]" on macOS), where the number corresponds to the tab's position.

This direct tab navigation is already common in many browsers and applications, allowing users to quickly jump to a specific tab without cycling through others. Having this functionality in addition to "Ctrl + Tab" would provide a faster and more efficient way to manage multiple open tabs.

**Comment by logotip4ik** [comment]
Isn't it already supported?
